src/components/SeriesNav/SeriesNavTileContent.js

```diff
@@ -81,8 +92,13 @@ const SeriesNavTileContent = ({
   if (inline) {
     return (
       <LinkContainer>
-        <p {...styles.label} {...colorScheme.set('color', 'text')}>
-          {`${current ? 'Sie lesen: ' : ''}${episode.label}`}
+        <p
+          {...styles.label}
+          {...(current ? styles.labelMedium : styles.labelRegular)}
+          {...colorScheme.set('color', 'text')}
+        >
+          {current ? `${t('styleguide/SeriesNav/current')} ` : null}
+          {episode.label}
```

Comment:
`{current ? '' : null}` was causing a annoying server/client html diff warning (react bug), let's use a single string:

```suggestion
          {`${current ? `${t('styleguide/SeriesNav/current')} ` : ''}${episode.label}`}
```

Reply:
done